src/app/notifications/notifications-page/notifications-page.component.ts

```diff
@@ -37,6 +50,22 @@ export class NotificationsPageComponent implements OnInit {
         console.log(error);
       }
     });
+
+    this.getNotificationsEnabled();
+  }
+
+  getNotificationsEnabled() {
+    this.profileService.getProfile()
+      .subscribe(profile => this.notificationsEnabled = profile.notificationsEnabled ?? true);
+  }
+
+  onNotificationsEnabledToggle($event: MatSlideToggleChange) {
+    this.accountService.toggleNotificationsEnabled()
+      .subscribe(account => {
+        this.notificationsEnabled = account.notificationsEnabled ?? true;
+        this.snackBar.open(account.notificationsEnabled
+          ? "Notifications have been enabled"
+          : "Notifications have been disabled", "OK", {duration: 3000});
```

Comment:
`SharedService` already does exactly this.
```suggestion
        this.sharedService.openSnack(account.notificationsEnabled
          ? "Notifications have been enabled"
          : "Notifications have been disabled", "OK", {duration: 3000});
```